assets/src/apps/authoring/components/AdaptiveRulesList/AdaptiveRulesList.tsx

```diff
@@ -467,8 +468,14 @@ const IRulesList: React.FC<any> = (props: any) => {
                         value={ruleToEdit.name}
                         onClick={(e) => e.preventDefault()}
                         onChange={(e) => setRuleToEdit({ ...rule, name: e.target.value })}
-                        onFocus={(e) => e.target.select()}
-                        onBlur={() => handleRenameRule(rule)}
+                        onFocus={(e) => {
```

Comment:
If user is trying to delete a rule name, we should not trigger the part component delete prompt.